recipes/rapidjson/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
```

Comment:
```suggestion
from conans import ConanFile, CMake
from conan.tools.build import cross_building
```
Linter does not accept old cross_building

Reply:
it's allowed in test_v1_package, conan v2 linter doesn't check this folder